SpatialGDK/Source/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -296,27 +297,14 @@ bool USpatialActorChannel::ReplicateActor()
 			//check(!Actor->IsFullNameStableForNetworking() || Interop->CanSpawnReplicatedStablyNamedActors());
 
 			// When a player is connected, a FUniqueNetIdRepl is created with the players worker ID. This eventually gets stored
-			// inside APlayerState::UniqueId when UWorld::SpawnPlayActor is called. If this actor channel is managing a pawn or a 
-			// player controller, get the player state.
+			// inside APlayerState::UniqueId when UWorld::SpawnPlayActor is called.
 			FString PlayerWorkerId;
-			APlayerState* PlayerState = Cast<APlayerState>(Actor);
```

Comment:
I've altered this code to be in its own function after adding retries for create entity. I think you'll need to update that code and potentially could remove that function.